internal/lsp/server.go

```diff
@@ -759,7 +759,10 @@ func (s *Server) handleCompletion(ctx context.Context, params *lsproto.Completio
 		params.Position,
 		params.Context,
 		getCompletionClientCapabilities(s.initializeParams),
-		&ls.UserPreferences{})
+		&ls.UserPreferences{
```

Comment:
Long term I think we're going to want to pass this info down a different way but this is fine for now of course.